### ISSUE TITLE
errorCallback called right after successCallback in Google Chrome and Edge

### ISSUE THREAD

**Issue by nikola1970** [maxmantz/redux-oidc#184]
Having this weird issue with Google Chrome (newest, tried Incognito as well) and Edge - after a successful login and successCallback it immediately calls errorCallback with message "ErrorResponse: login_required". No idea why but everthing works fine in FireFox (non chromium browser).

Anyone faced same issue? Any help? Thanks! :)

**Comment by SaimirKapaj** [comment]
We are facing the same issue.

**Comment by VahidMhrb** [comment]
Hi, same issue here.

**Comment by VahidMhrb** [comment]
Hi, @SaimirKapaj and @nikola1970 are you found a solution  ?

**Comment by rmlevangelio** [comment]
Same issue with me. But try to set the `monitorSession` to `false`.

**Comment by SaimirKapaj** [comment]
@VahidMhrb yes I found a solution. I'm using another library instead. Check https://github.com/bjerkio/oidc-react
